frontend/src/lib/taxonomy.tsx

```diff
@@ -20,21 +43,11 @@ const eventToPersonProperties = new Set([
     '$os_version',
     '$referring_domain',
     '$referrer',
-    // campaign params - automatically added by posthog-js here https://github.com/PostHog/posthog-js/blob/master/src/utils/event-utils.ts
-    'utm_source',
-    'utm_medium',
-    'utm_campaign',
-    'utm_content',
-    'utm_name',
-    'utm_term',
-    'gclid',
-    'gad_source',
-    'gbraid',
-    'wbraid',
-    'fbclid',
-    'msclkid',
+    ...CAMPAIGN_PROPERTIES,
 ])
 
+const SESSION_PROPERTIES_ADAPTED_FROM_EVENT = new Set(['$referrering_domain', ...CAMPAIGN_PROPERTIES])
```

Comment:
Hmm - this is not automatic, the sessions table doesn't automatically get new columns based on additions to `CAMPAIGN_PROPERTIES`

Reply:
I'd probably keep the session properties as a separate list